### ISSUE TITLE
Cant Download

### ISSUE THREAD

**Issue by SinkFriend** [pygame/pygame#4343]
im trying to download pygame on windows 11 but its just a tar file with a bounch of files

**Comment by DarthKitten2130** [comment]
Aren't you using pip??

**Comment by SinkFriend** [comment]
YES I CANT GET IT IT WONT OPEN PLS HELP I WANT TO MAKE MY GD FANGAME

**Comment by SinkFriend** [comment]
wait no im using pthon

**Comment by SinkFriend** [comment]
reply

**Comment by SinkFriend** [comment]
sent i

**Comment by DarthKitten2130** [comment]
are you using the pip package manager for python, I meant. If you don't know how to use pip, [read this](https://realpython.com/what-is-pip/).

A word of advice. If you don't even know how to use pip yet, I suggest you master the basics of python before delving into a PyGame project. Otherwise you'll just get more confused. Good luck!

**Comment by SinkFriend** [comment]
no its solved copilot helped me and i fanily can post games
